themes/scss/map/overlays.scss

```diff
@@ -344,6 +347,11 @@ $maxLegendContainerHeight: 300px;
   background: rgba(0, 0, 0, 0.08);
   content: '';
 }
+
+.Bubble-item--custom::before {
```

Comment:
The thing is we need to show/hide first and last item if custom labels are present and any of them is not empty. The last item is styled as part of the list container and the rest of the item are styled as list items, I couldn't find a way to achieve what you're saying. Do you have something in mind?